### PR TITLE
Because I said so thats why

### PR DESCRIPTION
**Pull request by Madeline-Ellingson**

### CODE REVIEW

File: Wine.py [hunk 1]
```diff
@@ -4,31 +4,48 @@
 """
 from statistics import mean
 
-num_of_cards = 0
-num_of_items = 0
-prices = []
+num_of_cards = 17
+num_of_items = 8
+prices_but_card = [
```
Comment (dmoore3554) [inline Wine.py:9]: This is awful

**Review by dmoore3554 — CHANGES_REQUESTED** [review]
git gud

File: Wine.py (hunk 1 above)
Comment (Madeline-Ellingson) [reply Wine.py:9]: You are aweful

**Comment by Madeline-Ellingson** [comment]
Fuck you.

**Pull request merged**